### ISSUE TITLE
Auto delete un-sellable junk is no longer working

### ISSUE THREAD

**Issue by eFse7en** [arkayenro/arkinventory#1409]
"Config->General->Junk->[V] Delete" option is no longer working (at least since Shadowlands).
It throws an error when trying to delete the Hearthstone:

3x [ADDON_ACTION_BLOCKED] AddOn 'ArkInventory' tried to call the protected function 'resume()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function \`resume'
[string "@ArkInventory\ArkInventory-30945.lua"]:11187: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>

After disabling this option I no longer get errors when selling junk, but I have to delete the hearthstone manually.

Version: **3.09.45**

**Comment by arkayenro** [comment]
it looks like blizzard protected that function and you can only delete items via a hardware event, ie you have to hit enter or click on ok after typing in delete

its impacting all mods that clean up

**Comment by eFse7en** [comment]
would it be possible to trigger that delete popup as a workaround?

**Comment by arkayenro** [comment]
i dont think so, im not stopping the popup in any way, im just calling the delete function.

id have to code my own popup in so i dont think it will work but i'll have a play around later to see as its not a pure code restriction, it now requires a hardware event (clicking a button or pressing a key) to sort of approve it, so theres a small chance ic an get something that might work.
